admin_manual/installation/nginx-root.conf.sample

```diff
@@ -79,7 +79,6 @@ server {
     add_header X-Frame-Options                   "SAMEORIGIN"        always;
     add_header X-Permitted-Cross-Domain-Policies "none"              always;
     add_header X-Robots-Tag                      "noindex, nofollow" always;
-    add_header X-XSS-Protection                  "1; mode=block"     always;
```

Comment:
Should add that to the "critical changes" update manual for admins

Reply:
IMO it is not problem if admins just leave the header inside. But I will add the note.